### ISSUE TITLE
Timer Dial 

### ISSUE THREAD

**Issue by mikwei** [fastaddons/GroupSpeedDial#261]
When Alarm or Timer expire it pops-up a window of the group.

What's needed is say timer pops-up dialog box with Timer (initial time) has completed/expired, or a URL/Text msg is opened.

Ditto for alarm,  "Alarm for N:NN has expired"

**Comment by Juraj-Masiar** [comment]
Yeah, I can't believe I've released it in current state :), there must have been some time pressure.
I'll try to improve it next week.
